packages/fast-components-styles-msft/src/label/index.ts

```diff
@@ -0,0 +1,20 @@
+import { IDesignSystem } from "../design-system";
+import { ComponentStyles } from "@microsoft/fast-jss-manager";
+import { ILabelClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { toPx } from "../utilities/units";
+import { typeRamp } from "../utilities/typography";
+
+const styles: ComponentStyles<ILabelClassNameContract, IDesignSystem> = {
+    label: {
+        fontSize: toPx(typeRamp.t8.vp3.fontSize),
+        display: "inline-block",
+        marginTop: toPx(21),
+        paddingBottom: toPx(7),
+        lineHeight: toPx(20),
+        "& + input": {
```

Comment:
I don't think we need this anymore. By default I think we'll have margin set to 0 on all components unless they are explicitly part of another components contract.

Reply:
Yep - changing per our discussion